ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/completion/EmptyLedgerIT.scala

```diff
@@ -44,38 +71,118 @@ class EmptyLedgerIT
       implicitPartyAllocation = false
     )
 
-  private[this] val applicationId = getClass.getSimpleName
+  "CommandCompletionService gives sensible ledger end on an empty ledger" in {
+    val lid = ledgerId()
+    val party = "partyA"
+    val completionService = CommandCompletionServiceGrpc.stub(channel)
+    for {
+      end <- completionService.completionEnd(CompletionEndRequest(lid.unwrap))
+      completions <- completionsFromOffset(
+        completionService = completionService,
+        ledgerId = lid,
+        parties = List(party),
+        offset = end.getOffset,
+        completionTimeout = 2.seconds,
+      )
+    } yield {
+      end.getOffset.value.absolute.get shouldBe ApiOffset.begin.toHexString
+      completions shouldBe empty
+    }
+  }
+
+  "ConfigManagementService accepts a configuration when none is set" in {
```

Comment:
IMO test should move from the `.services.completion` package to `.services.configuration`

Reply:
Will do.